docs/markdown/Syntax.md

```diff
@@ -186,6 +186,17 @@ As can be seen, the formatting works by replacing placeholders of type
 Strings also support a number of other methods that return transformed
 copies.
 
+#### .replace()
+
+Since 0.58.0, you can now replace characters in a string.
+
+```meson
+# Replaces all instances of one substring with another
+s = 'semicolons;as;separators'
+s = s.replace(';', ' ')
```

Comment:
I would rather use an example that makes it cleared it's replacing a substring and not characters.